docs/docs/behaviors/index.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: Behavior Overview
+sidebar_label: Overview
+---
+
+# Behavior Overview
+
+"Behaviors" are bindings that are assigned to and triggered by key positions on keymap layers, sensors (like an encoder) or combos. They describe what happens e.g. when a certain key position is pressed or released, or an encoder triggers a rotation event. They can also be recursively invoked by other behaviors, such as macros.
+
+Below is a summary of pre-defined behavior bindings and user-definable behaviors available in ZMK, with references to documentation pages describing them.
+
+## Key press behaviors
+
+| Binding       | Behavior                    | Description                                                                                                                                                                                                                                       |
+| ------------- | --------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `&kp`         | [Key Press](key-press.md)   | Send keycodes to the connected host when a key is pressed                                                                                                                                                                                         |
+| `&mt`         | [Mod Tap](mod-tap.md)       | Sends a different keypress, depending on whether it is held or tapped                                                                                                                                                                             |
+| `&gresc`      | [Mod Morph](mod-morph.md)   | Sends a different keypress, depending on whether a specified modifier is being held during the keypress                                                                                                                                           |
```

Comment:
```suggestion
| `&gresc`      | [Grave Escape](mod-morph.md#behavior-binding)   | Sends Grave Accent `\`` keycode if shift or GUI is held, sends Escape keycode otherwise                                                                                                                                           |
```
Describing grave-escape here specifically since that's the binding. (Need to double check whether the escaping worked)

Reply:
Changed the escaping to use of spaces (which markdown removes) in order to get `` ` `` to display.